Content.Shared/Electrocution/SharedElectrocutionSystem.cs

```diff
@@ -23,6 +23,19 @@ public void SetInsulatedSiemensCoefficient(EntityUid uid, float siemensCoefficie
             Dirty(uid, insulated);
         }
 
+        /// <summary>
+        /// Sets electrified value of component and marks dirty if required.
+        /// </summary>
+        public void SetElectrified(Entity<ElectrifiedComponent> ent, bool value)
+        {
+            var oldValue = ent.Comp.Enabled;
+            ent.Comp.Enabled = value;
+            if (value != oldValue)
+            {
+                Dirty(ent, ent.Comp);
+            }
+        }
```

Comment:
You did it here but also this is written in a weird way, just check if the values are the same before assigning (not sure if the JIT even accounts for this?)

Reply:
fixed.